pkg/tweetnacl/Makefile

```diff
@@ -0,0 +1,11 @@
+PKG_NAME=tweetnacl
+PKG_URL=https://github.com/RIOT-OS/tweetnacl
+PKG_VERSION=7ea05c7098a16c87fa66e9166ce301666f3f2623
```

Comment:
Euh... Yes, of course, but actually there's no official licence for this... it only says "TweetNaCl is a self-contained public-domain C library" so what you would suggest?